Gemfile

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+gem 'jekyll', '~> 4.0'
```

Comment:
As stated in the PR body, Jekyll is still responsible for the overarching build of the site. This is true of the GitHub Pages version of the site too, but was being handled transparently by GitHub.